test/pylib/suite/base.py

```diff
@@ -524,8 +524,10 @@ def prepare_dir(dirname: pathlib.Path, pattern: str) -> None:
 def prepare_dirs(tempdir_base: pathlib.Path, modes: list[str], gather_metrics: bool) -> None:
     prepare_dir(tempdir_base, "*.log")
     setup_cgroup(gather_metrics)
-    shutil.rmtree(tempdir_base / "ldap_instances", ignore_errors=True)
-    prepare_dir(tempdir_base / "ldap_instances", "*")
+    for directory in ['report', 'ldap_instances']:
+        full_path_directory = tempdir_base / directory
+        shutil.rmtree(full_path_directory, ignore_errors=True)
+        prepare_dir(full_path_directory, '*')
```

Comment:
(I leave this comment here, but it's not about the change itself)

The patch comment is repeating itself with no gain :)

> test.py: add common directory for reports
> Add common directory for reports for all modes

I believe most of reviewers understand that you "add common directory for reports" from the first try

Reply:
I just noticed that this patters is present in pretty much every patch of this PR